portfolio/src/main/webapp/hobbies.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="navBarLogin()">
```

Comment:
If this is something you want to happen on every page, you could register a listener at the bottom of script.js:

```js
window.addEventListener('DOMContentLoaded', (event) => {
    navBarLogin();
});
```

This will run in addition to anything in `body.onload()`, so you can leave `map.html` as-is.